### DISCUSSION TITLE
Cannot set time / clock source

### DISCUSSION THREAD

**Discussion by loopy321** [tagyoureit/nodejs-poolController#422] [Q&A] [answered]
Thanks for the awesome work!!  

The time on dashpanel and automations are off and I cannot figure out how to set it to the correct value.

I have verified that the system running the nodejs-poolcontroller has the correct time.  I tried changing the ClockSource in dashpanel, but it remains on "manual" despite trying to change it.

I see in the logs that it changes the value:
`0|njsPC      | [4/20/2022, 8:37:35 PM] info: [8:37:35 PM] 192.168.2.151 PUT /config/general {"options":{"units":0,"adjustDST":true,"clockMode":12,"clockSource":"server"},"location":{"timeZone":133}}
`
but when I reload dashpanel it always shows clocksource is "manual" and the time never gets fixed.

Any suggestions on how to fix the clock?

This is nodejspc 7.6.1.

**Comment by rstrouse** [comment]
What system are you running.  Nixie, IntelliTouch, EasyTouch, or IntelliCenter?

**Reply by loopy321** [reply]
EasyTouch 2 8.


_0|njsPC    | [4/20/2022, 10:04:06 PM] info: EasyTouch system config complete.
0|njsPC    | [4/20/2022, 10:04:06 PM] info: Initializing Nixie Controller
0|njsPC    | [4/20/2022, 10:04:06 PM] info: Nixie Controller Initialized_

**Reply by rstrouse** [reply]
There is only a manual setting on EasyTouch but we do a clever thing to set the time when it differs from the os time.  The Dallas clock on the OCP has notoriously bad drift.  Can I get you to upload a replay that includes changing the clock settings.

**Reply by loopy321** [reply]
[replay_mod.zip](https://github.com/tagyoureit/nodejs-poolController/files/8527047/replay_mod.zip)

**Reply by loopy321** [reply]
Rebooted the system and re-did the replay:
[replay.2.zip](https://github.com/tagyoureit/nodejs-poolController/files/8531828/replay.2.zip)

**Comment by rstrouse** [comment, marked as answer]
Go ahead and pull njsPC.  I do not have an EasyTouch but I suspect I found where it was jumping over that value.

**Reply by loopy321** [reply]
That seems to have allowed me to  set it to server and after some time, the time corrected.

Thank you!
